### PR TITLE
Support negative correlativity

### PR DESCRIPTION
**Pull request by darionyaphet**

### CODE REVIEW

File: src/common/interface/meta.thrift [hunk 1]
```diff
@@ -376,8 +376,10 @@ struct StatisItem {
     // The number of edges of current space
     4: i64                                    space_edges,
     5: map<common.PartitionID, list<Correlativity>>
-        (cpp.template = "std::unordered_map") part_corelativity,
-    6: JobStatus                              status,
+        (cpp.template = "std::unordered_map") positive_part_corelativity,
```
Comment (bright-starry-sky) [inline src/common/interface/meta.thrift:379]: Is the typo error of `corelativity` ?

**Comment by bright-starry-sky** [comment]
Could give some comments for `positive_part_corelativity` and `negative_part_corelativity` ?

File: src/common/interface/meta.thrift (hunk 1 above)
Comment (darionyaphet) [reply src/common/interface/meta.thrift:379]: It's too awesome.  it should be `correlativity`

**Comment by darionyaphet** [comment]
> Could give some comments for `positive_part_corelativity` and `negative_part_corelativity` ?

Yes, these two parameters need to be explained

**Review by bright-starry-sky — APPROVED** [review]
LGTM

**Review by panda-sheep — DISMISSED** [review]
Good

**Review by panda-sheep — APPROVED** [review]

**Review by liuyu85cn — APPROVED** [review]

**Review by bright-starry-sky — APPROVED** [review]
Nice

**Pull request merged**